src/framework/theme/components/calendar-kit/services/calendar-year-model.service.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { NbNativeDateService, NbCalendarYearModelService } from '@nebular/theme';
+
+describe('NbCalendarYearModelService', () => {
+  let yearModelService: NbCalendarYearModelService<Date>;
+  let yearsInView: number;
+  let yearsInRow: number;
+
+  beforeEach(() => {
+    yearModelService = new NbCalendarYearModelService(new NbNativeDateService('en-US'));
+    yearsInView = yearModelService.getYearsInView();
+    yearsInRow = yearModelService.getYearsInRow();
+  });
+
+  it('should return correct years range', () => {
```

Comment:
Would you mind to add more tests to cover corner cases? I'm dead sure that one test for such functionality is not enough

Reply:
I tried but can't come up with test cases. Share your ideas and I'll add more tests.

Reply:
Done in 585df46